pysmt/simplifier.py

```diff
@@ -664,3 +666,73 @@ def walk_div(self, formula, args, **kwargs):
         return self.manager.Div(sl, sr)
 
 # EOC Simplifier
+
+
+class BddSimplifier(Simplifier):
```

Comment:
This class will throw an exception at construction time if pyCUDD is not installed, it is OK for now, but for the next simplifier we would need a factory class...